src/public/target/sections/RelatedTargets/Section.js

```diff
@@ -59,65 +61,52 @@ const columns = (symbol, maxCountAOrB) => [
 
 const pageSize = 10; // unlikely to change for now, so no need to go into state
 
-class Section extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      pageIndex: 0,
-    };
-  }
+const Section = props => {
+  const [pageIndex, setPageIndex] = useState(0);
 
-  componentDidUpdate(prevProps, prevState) {
+  useEffect(() => {
     // fetchMore is Apollo's pagination function
     // no table sorting / ordering at the moment
-    const { fetchMore } = this.props;
-    const { pageIndex } = this.state;
-    if (prevState.pageIndex !== pageIndex) {
-      fetchMore({
-        variables: { index: pageIndex, size: pageSize },
-        updateQuery: (prev, { fetchMoreResult }) => {
-          if (!fetchMoreResult) {
-            return prev;
-          }
-          return Object.assign({}, prev, fetchMoreResult);
-        },
-      });
-    }
-  }
+    const { fetchMore } = props;
+    fetchMore({
+      variables: { index: pageIndex, size: pageSize },
+      updateQuery: (prev, { fetchMoreResult }) => {
+        if (!fetchMoreResult) {
+          return prev;
+        }
+        return Object.assign({}, prev, fetchMoreResult);
+      },
+    });
+  });
```

Comment:
This could be shortened a bit. Take a look at: https://github.com/opentargets/platform-app/blob/alpha/src/public/disease/sections/RelatedDiseases/Section.js#L64

But not really necessary.